GuiTest/src/main/java/io/github/cottonmc/test/TestDescription.java

```diff
@@ -42,7 +43,8 @@ public TestDescription(ScreenHandlerType<?> type, int syncId, PlayerInventory pl
 		buttonB.setOnClick(() -> slot.setIcon(new TextureIcon(new Identifier("libgui-test", "saddle.png"))));
 
 		root.add(buttonB, 5, 3, 4, 1);
-		root.add(new WButton(Text.literal("Button C")), 0, 5, 4, 1);
+		TextureIcon testIcon = new TextureIcon(new Texture(new Identifier("libgui-test", "icon.png")));
+		root.add(new WButton(testIcon, Text.literal("Button C")), 0, 5, 4, 1);
```

Comment:
For the examples, maybe it would make sense to create an ModItem each, per Widget.
Like a WButtonItem where a simple GUI with only Buttons is constructed and different settings are displayed?

Reply:
There's the `/libgui` client command which is probably good for this, it currently has test screens for tabs, scroll panels, insets and text fields:
https://github.com/CottonMC/LibGui/blob/f8c0715d2572fb4a74d308b5ad21d0868180fecb/GuiTest/src/main/java/io/github/cottonmc/test/client/LibGuiTestClient.java#L42-L49